plugins/performance-lab/tests/includes/site-health/audit-autoloaded-options/test-audit-autoloaded-options.php

```diff
@@ -187,6 +187,56 @@ static function () {
 		);
 	}
 
+	public function test_perflab_aao_autoloaded_options_auto_enable_functionality(): void {
+
+		$user_id = self::factory()->user->create( array( 'role' => 'administrator' ) );
+		wp_set_current_user( $user_id );
+
+		// Mock wp_redirect to avoid actual redirection.
+		add_filter(
+			'wp_redirect',
+			static function () {
+				return false;
+			}
+		);
```

Comment:
Where is a redirect happening?

```suggestion
		add_filter( 'wp_redirect', '__return_false' );
```

Reply:
In `perflab_aao_handle_update_autoload()` function.